src/clientagent/AstronClient.cpp

```diff
@@ -375,10 +407,12 @@ class AstronClient : public Client, public NetworkClient
             NetworkClient::send_disconnect();
         }
         break;
-        case CLIENT_OBJECT_SET_FIELD: {
```

Comment:
Looks like I let a quick formatting change into the PR. No, it is not related to the added functionality.